From 3b38593e4b62c7478f704c54ce404b56448572a0 Mon Sep 17 00:00:00 2001
From: Linu Cherian <linu.cherian@marvell.com>
Date: Tue, 4 Dec 2018 06:32:06 +0300
Subject: [PATCH 0847/1051] soc: octeontx2: Remove redundant mbox header clear
 in ptp

Response header reset is part of otx2_mbox_alloc_msg_rsp
and hence avoid clearing it again in ptp mbox handler.
Also it would corrupt already set response code.

Change-Id: Ia6dd6c999bc59c6db5374ca1b710142d4545b008
Fixes: 8e23c225d97b ("Add low level support for Marvell PTP coprocessor")
Signed-off-by: Linu Cherian <linu.cherian@marvell.com>
Reviewed-on: https://sj1git1.cavium.com/1754
Tested-by: sa_ip-sw-jenkins
Reviewed-by: Sunil Kovvuri Goutham <Sunil.Goutham@cavium.com>
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 drivers/soc/marvell/octeontx2/rvu_ptp.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/drivers/soc/marvell/octeontx2/rvu_ptp.c b/drivers/soc/marvell/octeontx2/rvu_ptp.c
index 2f7fafafa0cf..093ee093d68d 100644
--- a/drivers/soc/marvell/octeontx2/rvu_ptp.c
+++ b/drivers/soc/marvell/octeontx2/rvu_ptp.c
@@ -17,8 +17,6 @@ int rvu_mbox_handler_ptp_op(struct rvu *rvu, struct ptp_req *req,
 {
 	int err = 0;
 
-	memset(rsp, 0, sizeof(*rsp));
-
 	if (!rvu->ptp)
 		return -ENODEV;
 
-- 
2.17.1

